src/components/Forms/LoginForm.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="form-body">
+    <h1 class="auth-form-title form-body__title">Вход</h1>
+    <el-form ref="formInstance" :model="formModel" :rules="formRules" class="">
+      <el-form-item label="Почта" prop="email">
+        <el-input v-model="formModel.email" />
+      </el-form-item>
+      <el-form-item label="Пароль" prop="password">
+        <el-input v-model="formModel.password" type="password" />
+      </el-form-item>
+      <div class="d-f jc-sb ai-c">
+        <router-link :to="{ name: ROUTE_NAMES.RegistrationPage }">Регистрация</router-link>
+        <el-form-item class="mb-0">
+          <el-button type="primary" @click="sendLoginData">Войти</el-button>
```

Comment:
```suggestion
          <el-button type="primary" @click="handleUserSignIn">Войти</el-button>
```

Методы-обработчики именуются: handle{{цель}}{{действие}}. Пример: handleUserChange.